Server/JokesOnYou.Web.Api/Migrations/20210423023914_ChangedModelIntoId.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace JokesOnYou.Web.Api.Migrations
+{
+    public partial class ChangedModelIntoId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Jokes_AspNetUsers_AuthorId",
+                table: "Jokes");
+
+            migrationBuilder.DropIndex(
+                name: "IX_Jokes_AuthorId",
```

Comment:
Did you configure the FK in modelbuilder?

Reply:
no because its just a string and not a User anymore its not automatically linked to a FK ill try to change that.

Reply:
did i do it correct?